File: Problem110A.java
import java.util.Scanner;
public class Problem110A{
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        long n = input.nextLong();
        int C=0;
        while(n!=0)
        {
            long mod = n%10;
            if(mod==7 || mod==4)
            C++;
            n/=10;
        }
        if(C==7 || C==4)
        {
            System.out.println("YES");
        }
        else
        System.out.println("NO");
    }
}

File: PROBLEM 110A.py
n = int(input())
C=0
while n!=0:
    mod=n%10
    if mod==4 or mod==7:
        C=C+1
    n=n//10
if C==4 or C==7:
    print("YES")
else:
    print("NO")
exit(0)